public/homepage/scripts/main.js

```diff
@@ -33,10 +33,10 @@ require.config({
 function preloadBramble($) {
   var brambleHost = $("meta[name='bramble-host']").attr("content");
   [
-    brambleHost + "/dist/styles/brackets.min.css",
-    brambleHost + "/dist/bramble.js",
-    brambleHost + "/dist/main.js",
-    brambleHost + "/dist/thirdparty/thirdparty.min.js"
+    brambleHost + "dist/styles/brackets.min.css",
```

Comment:
this is specifically for brackets which has no locale in the url (until we localize it).